doc/skrub_pipeline.rst

```diff
@@ -395,3 +395,99 @@ Finally, there are other situations where using :func:`deferred` can be helpful:
   expressions on dataframes.
 - See :ref:`example_tuning_pipelines` for an example of hyper-parameter tuning using
   skrub pipelines.
+
+Subsampling
+~~~~~~~~~~~
+
+To speed-up the development of our pipeline, we can tell skrub to work on a
+subsample of our data.
+
+On any transformation that produces either a DataFrame, a Column (Series) or a
```

Comment:
This is very abstract to me. I wouldn't be able to link it to code that I can write.

We should probably say less here, and more in an example.